src/Containers/RightSideBar/Header.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import styled from 'styled-components';
+
+// STYLED COMPONENTS
+const Wrapper = styled.div`
+    text-align: center;
+`;
+const ModifierTools = styled.button`
+    font-family: Quicksand;
+    font-weight: bold;
+    background: #fff;
+    border: none;
+    padding: 17px;
+    display: inline-block;
+    font-size: 18px;
+    color: red;
+    border-bottom: 3px solid red;
+    outline: none;
+`;
+const Icon = styled.div`
+    margin-right: 3px;
+    display: inline-block;
+    -moz-border-radius: 8px/8px;
+    -webkit-border-radius: 8px 8px;
+    border-radius: 8px/8px;
+    border: solid 3px #f00;
+    width: 8px;
+    height: 8px;
+`;
+const CheckList = styled.button`
+    font-family: Quicksand;
+    font-weight: bold;
+    background: #fff;
+    border: none;
+    padding: 17px;
+    display: inline-block;
+    font-size: 18px;
+    color: red;
+    border-bottom: 3px solid red;
+    outline: none;
+`;
+// END OF STYLED COMPONENTS
+
+// MODIFIER PROPS
+export interface CheckHeaderProps {
+    checkTitle: string;
+    isGray: boolean;
+    onClick: React.MouseEventHandler<HTMLButtonElement>;
+}
+export interface ModifierHeaderProps {
+    modTitle: string;
+    isGray: boolean;
+    onClick: (e: React.MouseEvent) => void;
+}
+const onPage = {
+    color: 'gray',
+    borderBottom: 'none',
+};
+const IconGray = {
+    border: 'solid 3px gray',
+};
+// END OF MODIFIER PROPS
+
+export const CheckListHeader: React.FC<CheckHeaderProps> = ({
+    checkTitle,
+    isGray,
+    onClick,
+}): React.ReactElement => {
+    if (isGray) {
```

Comment:
The only thing different is the styling. Make this one component - no need for if, else block and pass in an optional prop. Look into building styled components using props and do this conditional styling